src/components/ValidatedInput/ValidFieldText.tsx

```diff
@@ -50,7 +50,8 @@ export interface ValidFieldTextProps
   dataQa?: string | null;
   dataQaProps?: string | null;
   validFieldTextRef?: React.RefObject<HTMLDivElement>;
-  rightIcon?: React.ReactNode;
+  prefixField?: React.ReactNode;
```

Comment:
for naming i more prefer to be `rightContent` and `leftContent`, smiliar with `rightIcon`

Reply:
I think prefix and suffix are more general, they implies before and after the content instead of left and right position